plugins/inputs/intel_pmt/README.md

```diff
@@ -0,0 +1,434 @@
+  pmt_source = "https://raw.githubusercontent.com/intel/Intel-PMT/master/xml/pmt.xml"
+  
+  ## Filter metrics by their datatype.
+  ## See the Filtering section in README for more details.
+  ## If empty, all metrics are collected.
+  # datatype_metrics = []
+  
+  ## Filter metrics by their name.
+  ## See the Filtering section in README for more details.
+  ## If empty, all metrics are collected.
+  # sample_metrics = []
+```
+
+## Example Configuration: `pmt_source` as a filepath
+
+This configuration uses a local copy of the XML specification:
+
+```toml
+[[inputs.intel_pmt]]
+  pmt_source = "/home/telegraf/Intel-PMT/xml/pmt.xml"
+```
+
+## Example Configuration: C-State residency and temperature with a datatype metric filter
+
+This configuration allows getting only a subset of metrics
+with the use of a datatype filter:
+
+```toml
+[[inputs.intel_pmt]]
+  pmt_source = "/home/telegraf/Intel-PMT/xml/pmt.xml"
+  datatype_metrics = ["tbandwidth_28b","ttemperature"]
+```
+
+## Example Configuration: C-State residency and temperature with a sample metric filter
+
+This configuration allows getting only a subset of metrics
+with the use of a sample filter:
+
+```toml
+[[inputs.intel_pmt]]
+  pmt_source = "/home/telegraf/Intel-PMT/xml/pmt.xml"
+  sample_metrics = ["C0Residency","C1Residency", "Cx_TEMP"]
+```
+
+## Prerequisites
+
+Minimum Linux kernel version 5.11 with
+
+- the `intel_pmt_telemetry` module loaded (on kernels 5.11-5.14)
+- the `intel_pmt` module loaded (on kernels 5.14+)
+
+Intel PMT is exposed on a limited number of devices, e.g.
+
+- 4th Generation Intel® Xeon® Scalable Processors
+(codenamed Sapphire Rapids / SPR)
+- 6th Generation Intel® Xeon® Scalable Processors
+(codenamed Granite Rapids / GNR)
+
+PMT space is located in `/sys/class/intel_pmt` with `telem` files requiring
+root privileges to read.
+
+### If Telegraf is not running as a root user
+
+By default, the `telem` binary file requires root privileges to be read.
+To avoid running Telegraf as a root, use one of these steps:
+
+- Add read permission for the current user to `telem` files exposed by the
+`intel_pmt` driver
+
+  ```sh
+  sudo chmod o+r /sys/class/intel_pmt/telem*/telem
+  ```
+
+- Add the following capabilities to the Telegraf executable
+
+  ```sh
+  sudo setcap cap_dac_read_search+ep /usr/bin/telegraf
+  ```
+
+## Metrics
+
+All measurements have the following tags:
+
+- `guid` (unique id of an Intel PMT space).
+- `numa_node` (NUMA node the sample is collected from).
+- `sample_name` (name of the gathered sample).
+- `sample_group` (name of a group to which the sample belongs).
+- `datatype_idref` (datatype to which the sample belongs).
+
+Measurements prefixed with `Cx_` where `x` is the core number also have the
+following tag:
+  
+- `core` (core to which the measurement relates).
+  
+Measurements prefixed with `CHAx_` where `x` is the CHA number also have the
+following tag:
+
+- `cha` (Caching and Home Agent to which the measurement relates).
```

Comment:
You use the term measurement here, but do you mean a measurement's field? An actual data point?

Reply:
This entire section describes a measurement as a single data point, which will have tags as described and a value.

Specifically saying: _Measurements prefixed with `CHAx_`_ I meant the sample_name

We can change this to `Measurements with sample_name prefixed with ...`

Reply:
I am not entirely sure that makes it more clear. We use "measurements" to mean the result of an input, which contains tags and fields. For this plugin, the measurement is called "intel_pmt".  Instead of measurement I would use "sample name" as that is the tag you are specifically talking about.

Reply:
Okay, so how about `Sample_name prefixed with ...`?

Reply:
Renamed `Measurement` to `Metric` and also changed to `sample_name prefixed in XMLs...`